<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>772</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Choose a sorting to simulate:</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bubbleSorterButton">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Bubble sorting</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="insertSorterButton">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Insert sorting</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="selectSorterButton">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Select sorting</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="heapSorterButton">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Heap sorting</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="quickSorterButton">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Quick sorting</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mergeSorterButton">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Merge sorting</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="countSorterButton">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Count sorting</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pushButton_1">
           <property name="maximumSize">
            <size>
             <width>45</width>
             <height>45</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Algerian</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="maximumSize">
            <size>
             <width>45</width>
             <height>45</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Algerian</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="maximumSize">
            <size>
             <width>45</width>
             <height>45</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Algerian</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>3</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="maximumSize">
            <size>
             <width>45</width>
             <height>45</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Algerian</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>4</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_5">
           <property name="maximumSize">
            <size>
             <width>45</width>
             <height>45</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Algerian</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>5</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_6">
           <property name="maximumSize">
            <size>
             <width>45</width>
             <height>45</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Algerian</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>6</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_7">
           <property name="maximumSize">
            <size>
             <width>45</width>
             <height>45</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Algerian</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>7</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_8">
           <property name="maximumSize">
            <size>
             <width>45</width>
             <height>45</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Algerian</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>8</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_9">
           <property name="maximumSize">
            <size>
             <width>45</width>
             <height>45</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Algerian</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>9</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_10">
           <property name="maximumSize">
            <size>
             <width>45</width>
             <height>45</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Algerian</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>10</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>30</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>You chose:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="simulateButton">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Launch simulating</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
